ptbcontrib/redis_persistence/redispersistence.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+#
+# A library containing community-based extension for the python-telegram-bot library
+# Copyright (C) 2020-2021
+# The ptbcontrib developers
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module implements RedisPersistence class."""
+
+from typing import Any
+import redis
+from telegram.ext import DictPersistence
+
+
+class RedisPersistence(DictPersistence):
+    """
+    Bot's persistence between restarts implemented via a Redis database.
+    Note: in the database, all keys created by this class are prefixed with 'ptb:'.
```

Comment:
Usually we have the same setup with the args, so one new line and indenting.

Maybe you want to allow users to change the prefixes?
```suggestion
    Note: 
        In the database, all keys created by this class are prefixed with 'ptb:'.
```